Pod/Classes/VOKCoreDataManager.m

```diff
@@ -93,7 +95,7 @@ - (NSManagedObjectContext *)tempManagedObjectContext
         return nil;
     }
 
-    NSManagedObjectContext *tempManagedObjectContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSConfinementConcurrencyType];
+    NSManagedObjectContext *tempManagedObjectContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSPrivateQueueConcurrencyType];
```

Comment:
I thought `NSPrivateQueueConcurrencyType` required the use of `performBlock:` (and related methods) as the context creates and manages its own private queue. To me, `NSConfinementConcurrencyType` meant we managed our queues manually.